test/kafo_wizards/highline/wizard_test.rb

```diff
@@ -4,7 +4,11 @@
 describe KafoWizards::HighLine::Wizard do
   let(:input) { StringIO.new }
   let(:output) { StringIO.new }
-  let(:name) { factory.string(:name, :default_value => 'Any') }
+
+  def name
```

Comment:
I think you should continue using `let` but with a different name. because now you're probably overriding built in minitest functionality.